### DISCUSSION TITLE
How to select points in UnstructuredGrid?

### DISCUSSION THREAD

**Discussion by fdinhSG** [pyvista/pyvista#4232] [Q&A] [answered]
I'm plotting a mesh with pyvista using [`UnstructuredGrid`](https://docs.pyvista.org/api/core/_autosummary/pyvista.UnstructuredGrid.html#pyvista-unstructuredgrid). It's going pretty well and I managed to plot the geometry I want.

I now want to access methods outside of `UnstructuredGrid`: for example, I want to be able to click and select on mesh elements and I see this is possible to do with the [`enable-point-picking`](https://docs.pyvista.org/api/plotting/_autosummary/pyvista.Plotter.enable_point_picking.html#enable-point-picking) method. The only problem is that this method does not belong to `UnstructuredGrid`, but to the `Plotter` class.

Does this mean I cannot use that method, or there are other workarounds?

Thanks!

**Comment by banesullivan** [comment, marked as answer]
You have to display the mesh in the plotter to then perform picking. Please take a look at some of the examples: https://docs.pyvista.org/examples/02-plot/point-picking.html#sphx-glr-examples-02-plot-point-picking-py

**Reply by fdinhSG** [reply]
Thanks! I missed the fact that you could add mesh points to a plotter.
